app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedActivityTest.kt

```diff
@@ -750,6 +750,116 @@ class MarkChaptersCompletedActivityTest {
     }
   }
 
+  @Test
+  fun testMarkChaptersCompletedActivity_clickMarkCompleted_activityFinishes() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+      assertThat(activityTestRule.activity).isNull()
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_configChange_clickMarkCompleted_activityFinishes() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+      assertThat(activityTestRule.activity).isNull()
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_selectMultipleChapters_clickMarkCompleted_reopenActivity_completedChaptersAreChecked() { // ktlint-disable max-line-length
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      scrollToPosition(position = 1)
+      performItemCheckOnRecyclerViewItemAtPosition(itemPosition = 1)
+      scrollToPosition(position = 4)
+      performItemCheckOnRecyclerViewItemAtPosition(itemPosition = 4)
+      scrollToPosition(position = 9)
+      performItemCheckOnRecyclerViewItemAtPosition(itemPosition = 9)
+      scrollToPosition(position = 10)
+      performItemCheckOnRecyclerViewItemAtPosition(itemPosition = 10)
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+    }
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      scrollToPosition(position = 1)
+      verifyItemCheckedOnRecyclerViewItemAtPosition(itemPosition = 1)
+      scrollToPosition(position = 4)
+      verifyItemCheckedOnRecyclerViewItemAtPosition(itemPosition = 4)
+      scrollToPosition(position = 9)
+      verifyItemCheckedOnRecyclerViewItemAtPosition(itemPosition = 9)
+      scrollToPosition(position = 10)
+      verifyItemCheckedOnRecyclerViewItemAtPosition(itemPosition = 10)
+    }
+  }
+
+  @Test
+  fun testMarkChaptersCompletedActivity_configChange_selectMultipleChapters_clickMarkCompleted_reopenActivity_completedChaptersAreChecked() { // ktlint-disable max-line-length
```

Comment:
Suggest writing it as `fun testMarkChapsCompleted_land_selectChapters_clickCompleted_reopen_completedChapsAreChecked() { ` 

as in long term we will remove all `// ktlint-disable max-line-length`

Reply:
Done.